nicer/sci_plots.py

```diff
@@ -63,12 +64,14 @@ def sci_plots(etable, gtitable, args):
 
     #tstart, tstop, exposure
     exposure = float(etable.meta['EXPOSURE'])
-    tstart = etable.meta['DATE-OBS'].replace('T',' at ')
-    tend = etable.meta['DATE-END'].replace('T', ' at ')
+    #tstart = etable.meta['DATE-OBS'].replace('T',' at ')
+    #tend = etable.meta['DATE-END'].replace('T', ' at ')
+    tstart = TimeDelta(etable.meta['TSTART'], format='sec')+Time(etable.meta['MJDREFI']+etable.meta['MJDREFF'], format='mjd')
```

Comment:
You might want to add scale='tt' to the Time() and TimeDelta(), since that is the correct timescale

Reply:
Done!